performance_regression_test.py

```diff
@@ -271,6 +271,18 @@ def _run_cql_commands(self, cmds, node=None):
             with self.db_cluster.cql_connection_patient(node) as session:
                 session.execute(cmd)
 
+    @latency_calculator_decorator
+    def steady_state_latency(self, sleep_time):  # pylint: disable=no-self-use
```

Comment:
I think better name would be `measurement_time` instead of `sleep_time`

Reply:
changed to `steady_state_time`